api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -695,17 +697,56 @@ public void setProfessionalUrl(String professionalUrl) {
     this.professionalUrl = professionalUrl;
   }
 
-  @OneToOne(
+  /**
+   * The mapping from user:address is logically 1:1 in the app, but is modeled as 1:many in the DB.
+   * As a workaround until we're ready to correrct the constraint in the schema, I've added a
+   * binding to a Set to allow multiple addresses to be stored and also exposed a transient setter
+   * and getter for a single address.
+   *
+   * @return
+   */
+  @OneToMany(
       cascade = CascadeType.ALL,
       orphanRemoval = true,
       fetch = FetchType.LAZY,
       mappedBy = "user")
-  public DbAddress getAddress() {
-    return address;
+  public Set<DbAddress> getAddresses() {
+    return addresses;
+  }
+
+  public void setAddresses(Set<DbAddress> address) {
```

Comment:
Should this method be commented with a stern warning that, even though the DbUser class holds multiple addresses, the application actually treats this as a one-to-one mapping, so this method should probably never be called?

Reply:
Yeah... DbUser has  no opinion on the matter; if a PM wants multiple addresses tomorrow, we're ready, but a comment can't hurt.